<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BaseClass</class>
 <widget class="QMainWindow" name="BaseClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1330</width>
    <height>787</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Base</string>
  </property>
  <property name="windowIcon">
   <iconset resource="Base.qrc">
    <normaloff>:/Resources/icon.ico</normaloff>:/Resources/icon.ico</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QWidget" name="widget" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QWidget" name="widget_2" native="true">
         <layout class="QGridLayout" name="gridLayout">
          <item row="3" column="0">
           <widget class="QLineEdit" name="name"/>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Фамилия</string>
            </property>
           </widget>
          </item>
          <item row="7" column="1">
           <widget class="QLineEdit" name="tel2"/>
          </item>
          <item row="7" column="0">
           <widget class="QLineEdit" name="surName"/>
          </item>
          <item row="4" column="1">
           <widget class="QLineEdit" name="tel1"/>
          </item>
          <item row="3" column="2">
           <widget class="QLineEdit" name="birth"/>
          </item>
          <item row="3" column="1">
           <widget class="QLineEdit" name="tel0"/>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Имя</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Телефоны</string>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Дата рождения (&lt;день&gt;.&lt;месяц&gt;)</string>
            </property>
           </widget>
          </item>
          <item row="4" column="2">
           <widget class="QPushButton" name="clearAllButton">
            <property name="text">
             <string>Очистить все</string>
            </property>
           </widget>
          </item>
          <item row="7" column="2">
           <widget class="QPushButton" name="exportButton">
            <property name="text">
             <string>Экспортировать в .xlsx</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="addButton">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>100</height>
          </size>
         </property>
         <property name="text">
          <string>Добавить</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QTableWidget" name="tableWidget">
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">border-color: rgb(116, 116, 116);</string>
      </property>
      <property name="alternatingRowColors">
       <bool>true</bool>
      </property>
      <property name="sortingEnabled">
       <bool>false</bool>
      </property>
      <property name="cornerButtonEnabled">
       <bool>false</bool>
      </property>
      <attribute name="horizontalHeaderVisible">
       <bool>true</bool>
      </attribute>
      <attribute name="horizontalHeaderCascadingSectionResizes">
       <bool>false</bool>
      </attribute>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>150</number>
      </attribute>
      <attribute name="horizontalHeaderMinimumSectionSize">
       <number>10</number>
      </attribute>
      <attribute name="horizontalHeaderStretchLastSection">
       <bool>true</bool>
      </attribute>
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
      <column>
       <property name="text">
        <string>Имя</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Фамилия</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Телефон #1</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Телефон #2</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Телефон #3</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Дата рождения</string>
       </property>
      </column>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>name</tabstop>
  <tabstop>surName</tabstop>
  <tabstop>tel0</tabstop>
  <tabstop>tel1</tabstop>
  <tabstop>tel2</tabstop>
  <tabstop>birth</tabstop>
  <tabstop>addButton</tabstop>
  <tabstop>clearAllButton</tabstop>
  <tabstop>exportButton</tabstop>
  <tabstop>tableWidget</tabstop>
 </tabstops>
 <resources>
  <include location="Base.qrc"/>
 </resources>
 <connections/>
</ui>
